tools/git-change-exec/Makefile

```diff
@@ -0,0 +1,10 @@
+# Copyright(c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+git-change-exec: main.go actions.go
+	go build
```

Comment:
Will it create some files that should go under `.gitignore`?

Reply:
you mean https://github.com/lf-edge/eve/pull/4202/files#diff-c9d800638e653a196bb171cf4067cc973c55ed6f55a29ff1f26510589bd57f54 ?

Reply:
Sorry, I missed it. Expected `a.out` or something. Joking.